### PR TITLE
updated Step 4 in README to ICP4D 3.0

### PR DESCRIPTION
**Pull request by ljbennett62**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,15 +216,15 @@ You will see the triangle with `!` under `Fairness` -> `Sex`. This indicates tha
 
 *Explanations show the most significant factors when determining an outcome. Classification models also include advanced explanations. Advanced explanations are not available for regression, image, and unstructured text models.*
 
-* Click on the info icon next to `Minimum changes for No Risk outcome` and look at the feature values:
+* Click on the info icon next to `Minimum changes for another outcome` and look at the feature values:
 
 *Pertinent Negative
-If the feature values were set to these values, the prediction would change. This is the minimum set of changes in feature values to generate a different prediction. Each feature value is changed so that it moves towards its median value in the training data.*
+Pertinent Negatives (PN) are feature values obtained by changing the value of each feature away from its median such that the model prediction changes. If the feeature attributes were set to these values, the prediction would change. This is the minimum set of changes in feature values to generate a different predictioon.*
```
Comment (markstur) [inline README.md]: typos "feeature" and "predictioon"

File: README.md [hunk 2]
```diff
@@ -217,15 +216,15 @@ You will see the triangle with `!` under `Fairness` -> `Sex`. This indicates tha
...
 
 * Click on the info icon next to `Maximum changes allowed for the same outcome` and look at the feature values:
 
 *Pertinent Positive
-The prediction will not change even if the feature values are set to these values. This is the maximum change allowed while maintaining the existing prediction. Each feature value is changed so that it moves towards its median value in the training data.*
+Pertinent Postives (PP) are feature values obtained by changing the values of each feature towards its median such that thee model preediction remains the same. The prediction does not chang, even if the feature attributes are set to these values. This is the maximum possible change whille maintaining the existing prediction. Each feature value changes so that it moves towards it median value in the training data.*
```
Comment (markstur) [inline README.md]: typos "thee", "preediction", "chang", "whille" and "it median value" near the end should be "its median value"

File: README.md [hunk 3]
```diff
@@ -171,43 +171,42 @@ The *Insights Dashboard* provides an overview of the models that OpenScale is mo
...
 
 ![OpenScale Insight Dashboard Tile Open](doc/source/images/OpenScaleInsightDashTileOpen.png)
 
 Do not worry if the name you see does not match exactly with the screenshot. The deployment name you see will correspond to the variable used in the Jupyter notebook
 
-You will see the triangle with `!` under `Fairness` -> `Sex`. This indicates that there has been an alert for the `Fairness` monitor. Alerts are configurable, based on thresholds for fairness outcomes which can be set and altered as desired.
+![OpenScale Quality Monitor](doc/source/images/OpenScaleQualityMonitor.png)
 
-* By moving your mouse pointer over the graph, you can see the values change, and which contains bias. Click one spot to view the details. Later, we'll click `Configure Monitors` to get a Fairness endpoint:
+The quality monitor scans the requests sent to your model deployment \(i.e the payload\) to let you know how well your model predicts outcomes. Quality metrics are calculated hourly, when OpenScale sends manually labeled feedback data set to the deployed model.
```
Comment (markstur) [inline README.md]: "i.e" is missing a period after the e.  It is usually followed by a comma as well, but not sure that is better.
Not sure about the last sentence it just sounds a little off to me.  Changing "feedback data set" to just "feedback data" or just "feedback" would help.  Alternatively, if you want to keep "feedback data set" then I'd either make it plural "data sets" or keep it singular and just add "a" or before "manually labelled". Not sure if the grammar is wrong or if that is just my preference/style.

**Review by markstur — CHANGES_REQUESTED** [review]
Found some typos

File: README.md (hunk 2 above)
Comment (sandhya-nayak) [reply README.md]: fixed.

File: README.md (hunk 3 above)
Comment (sandhya-nayak) [reply README.md]: fixed.

**Review by markstur — APPROVED** [review]

**Pull request merged**